### PR TITLE
Add instructions of building bazel-stress

### PR DESCRIPTION
**Pull request by huangw5**

### CODE REVIEW

File: bazel-stress.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+First start the container in interactive mode:
+
+```
+docker run --rm -it \
+  -e GOOGLE_APPLICATION_CREDENTIALS=/path/to/rbe_credentials.json \
+  -v /path/to/rbe_credentials.json:/path/to/rbe_credentials.json \
+  bazel-rbe:latest bash
+```
+
+Then inside the container (Replace `your_rbe_project` and
```
Comment (martensryan) [inline bazel-stress.md]: s/Replace/replace/

File: bazel-stress.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+# Kick off the build remotely.
+bazel build --config=remote :all
+```
+
+# Run many bazel-stress builders in parallel
+
+We provide a convenient script
+[build-stress.sh](https://github.com/huangw5/tensorflow-rbe/blob/master/docker/build-stress.sh)
+in the docker container to run many bazel-stress in parallel. We use Google
```
Comment (martensryan) [inline bazel-stress.md:64]: Should we say Google Compute Engine instead of GCP?

File: bazel-stress.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+[official documentation](https://cloud.google.com/remote-build-execution/docs/access-control)
+of Google Remote Build Execution. Note that you will need to be whitelisted in
+order to access the documentation. See
+[the instructions](https://groups.google.com/forum/#!forum/rbe-alpha-customers).
+
+## Run bazel-stress builders in Managed Instance Group (MIG)
+
+### Step 1: Create an instance template
+
+1.  Visit https://pantheon.corp.google.com/compute/instanceTemplates/add Make
```
Comment (martensryan) [inline bazel-stress.md]: Missing period

**Review by martensryan — APPROVED** [review]

File: bazel-stress.md (hunk 1 above)
Comment (huangw5) [reply bazel-stress.md]: Done

File: bazel-stress.md (hunk 3 above)
Comment (huangw5) [reply bazel-stress.md]: Done

File: bazel-stress.md (hunk 2 above)
Comment (huangw5) [reply bazel-stress.md:64]: Done

**Pull request merged**